react-backend/client/src/components/visitors/signup_form.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import Input from './input';
+import Select from './select';
+
+export default () => (
+  <div>
+    <section className="Main col-9">
+      <h1>Please tell us about yourself</h1>
+      {error && (
+        <div className="ErrorText">
+          {error.map(el => <span key={el}>{el}</span>)}
+        </div>
+      )}
+      <form className="Signup" onChange={this.handleChange}>
+        <Input question="Name" option="fullname" />
+        <Input question="Email" option="email" />
+        <Input question="Phone Number (optional)" option="phone" />
+        <Select
+          question="Gender"
+          option="gender"
+          choices={['male', 'female', 'prefer not to say']}
+        />
+        <Select question="Year of Birth" option="year" choices={years} />
+      </form>
+      <button onClick={this.checkUserExists} className="Button">
+        {' '}
+        Next{' '}
+      </button>
+    </section>
+
+    <aside className="Privacy col-3">
+      <h3 className="Privacy__title">
+        Why are we collecting this information?
+      </h3>
+      <p className="Privacy__text">
+        Here at XXXXXXXX, we take your privacy seriously: we will only use your
+        personal information to administer your account to provide the products
+        and services you have requested from us, and improve how we deliver
+        those.
+        <br />
+        <br />
+        However, from time to time we would like to contact you with details of
+        other offers we provide. If you consent to us contacting you by email,
+        please tick to agree:{' '}
+      </p>
+      <form className="EmailCheckbox" onChange={this.handleChange}>
+        <Input type="checkbox" option="emailContact" checked />
```

Comment:
This isn't what I meant; I meant that the checkboxes and the other fields should all be part of the _same_ `form` element. You can nest any element you like inside a `form`, so you can put your `section` and `aside` within it.